README.md

```diff
@@ -16,6 +16,7 @@ If you are using one of the ports of lz-string to decode on the server what was
 ```shell
 $ npm install -g lz-string
 $ lz-string input.js > output.txt
```

Comment:
My preference would be to have `compress` and `decompress` as a pair of "commands".

The group seems to be ready to release a potentially BC in a new version, so having the CLI also have a BC is not a no-no.